srcpkgs/nextdns/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'nextdns'
+pkgname=nextdns
+version=1.32.1
+revision=1
+build_style=go
+go_import_path="github.com/nextdns/nextdns"
+short_desc="NextDNS CLI client (DoH Proxy)"
+maintainer="Anubhav Kini <anubhavkini@gmail.com>"
+license="MIT"
+homepage="https://nextdns.io/"
+distfiles="https://github.com/${pkgname}/${pkgname}/archive/refs/tags/v${version}.tar.gz"
+checksum=cab9a36ec8b8157b7e4988300f82fad164dda70e13108d86f6921fda87ce6501
+
+do_install() {
+	for f in ${GOPATH}/bin/* ${GOPATH}/bin/**/*; do
+		if [ -f "$f" ] && [ -x "$f" ]; then
+			vbin "$f"
+		fi
+	done
```

Comment:
This should not replicate the `build_style`s functions, instead use `pre_install` or `post_install` for `vsv` and `vlicense`.

Reply:
Fixed it, thanks.